shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -554,7 +554,34 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
     File dstFile = new File(dstPath.getPath());
     String randomSuffix =
         String.format(".%s_copyToLocal_", RandomStringUtils.randomAlphanumeric(8));
-    File tmpDst = new File(dstFile.getAbsolutePath() + randomSuffix);
+    File tmpDst;
+    File outputFile;
+    String dstAbsolute = dstFile.getAbsolutePath();
+    String fileName;
+    if (dstPath.getPath().equals("")) {
```

Comment:
@aaudiber  I think it is necessary to handle relative path. Bcz the `copyFromLocal` command also does not recognize relative path